### DISCUSSION TITLE
requestAnimationFrame is not defined Nuxt issue

### DISCUSSION THREAD

**Discussion by andriyg** [advanced-chat/advanced-chat-components#450] [Q&A]
Hey guys, I'm trying to use the chat in a Nuxt project and it fails on the first steps with "requestAnimationFrame is not defined" message, I've tried Nuxt 2 and 3 versions. Am I missing something?

**Comment by antoine92190** [comment]
Could you please share your project or even better a brand new nuxt project so I can reproduce on my side?

**Reply by andriyg** [reply]
https://github.com/andriyg/nuxt3app

**Comment by antoine92190** [comment]
Thanks I will look into it. In the meantime you can check this issue and see if it helps: https://github.com/antoine92190/vue-advanced-chat/issues/265

**Reply by antoine92190** [reply]
You can also check all related nuxt issues: https://github.com/antoine92190/vue-advanced-chat/issues?q=nuxt
There are solutions to make it work, but indeed Nuxt compatibility can be improved.

**Comment by antoine92190** [comment]
I just opened an issue with a working demo code: https://github.com/antoine92190/vue-advanced-chat/issues/451

**Reply by andriyg** [reply]
Thank you for the quick response Antoine!
